cdc/puller/sorter/heap_sorter.go

```diff
@@ -85,7 +86,7 @@ type heapSorter struct {
 func newHeapSorter(id int, out chan *flushTask) *heapSorter {
 	return &heapSorter{
 		id:        id,
-		inputCh:   make(chan *model.PolymorphicEvent, 1024*1024),
+		inputCh:   make(chan *model.PolymorphicEvent, sortHeapInputChSize),
```

Comment:
```suggestion
		inputCh:   make(chan *model.PolymorphicEvent, sortHeapInputChSize),
```
the `sortHeapInputChSize` is not 1024*1024, is it expected?

Reply:
Yes. 1024 * 1024 is too large.